tests/parquet_test.py

```diff
@@ -577,6 +580,7 @@ def test_segarray_string(self):
         words = ak.array(['one,two,three', 'uno,dos,tres'])
         strs, segs = words.split(',', return_segments=True)
         x = ak.SegArray(segs, strs)
+        # x = ak.SegArray(ak.array([0, 0, 2]), ak.array(["a", "b", "c", "d"]))
```

Comment:
Does this still need to be here?

Reply:
Nope it was left over from some extra confirmation I was doing. It has been removed.